internal/connect/collectors.go

```diff
@@ -4,24 +4,38 @@ import (
 	"github.com/SUSE/connect-ng/internal/collectors"
 )
 
-// Collectors which are to be used when fetching the system information.
-var usedCollectors = []collectors.Collector{
-	collectors.CPU{},
-	collectors.Hostname{},
-	collectors.Memory{},
-	collectors.UUID{},
-	collectors.Virtualization{},
-	collectors.CloudProvider{},
-	collectors.Architecture{},
-	collectors.ContainerRuntime{},
-
-	// Optional collectors
-	collectors.Uname{},
-	collectors.SAP{},
+// Fetch system information based on the available collectors for the system.
+func FetchSystemInformation(infoOnly bool) (collectors.Result, error) {
```

Comment:
Is `infoOnly` ever used? Same for the other function.

Reply:
no longer in FetchSystemInformation, it was when profile data was being passwd as hwinfo.